### PR TITLE
Removed usage of deprecated functions in draw2d core package

### PR DESCRIPTION
**Pull request by azoitl**
Addresses #21

### CODE REVIEW

File: org.eclipse.draw2d/src/org/eclipse/draw2d/ConnectionEndpointLocator.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2000, 2010 IBM Corporation and others.
+ * Copyright (c) 2000, 2023 IBM Corporation and others.
```
Comment (ptziegler) [inline org.eclipse.draw2d/src/org/eclipse/draw2d/ConnectionEndpointLocator.java:2]: Just a reminder, we're in 2024 already 😉

File: org.eclipse.draw2d/src/org/eclipse/draw2d/ConnectionEndpointLocator.java [hunk 2]
```diff
@@ -73,23 +72,22 @@ private int calculateConnectionLocation(Point loc, Point topLeft, Point center)
...
+		if (loc.y < center.y) {
+			return 3;
 		} else {
-			// Connection start point along top or bottom
-			if (loc.y < center.y) {
-				return 3;
-			} else {
-				return 1;
-			}
+			return 1;
```
Comment (ptziegler) [inline org.eclipse.draw2d/src/org/eclipse/draw2d/ConnectionEndpointLocator.java:90]: Is there a reason to keep the other "else" branches?

Comment (azoitl) [reply org.eclipse.draw2d/src/org/eclipse/draw2d/ConnectionEndpointLocator.java:90]: This was done by the auto clean-up. Now that I see it also the else in line 82 should be removed. Will fix this,.

File: org.eclipse.draw2d/src/org/eclipse/draw2d/ConnectionEndpointLocator.java (hunk 1 above)
Comment (azoitl) [reply org.eclipse.draw2d/src/org/eclipse/draw2d/ConnectionEndpointLocator.java:2]: It seems I'm still living in the past ;-) will fix it.

**Pull request merged**